Client/Frontend/Browser/ClipboardBarDisplayHandler.swift

```diff
@@ -118,41 +118,31 @@ class ClipboardBarDisplayHandler: NSObject, URLChangeDelegate {
         }
         return false
     }
-
+    
     func checkIfShouldDisplayBar() {
-        guard self.prefs.boolForKey("showClipboardBar") ?? false else {
-            // There's no point in doing any of this work unless the
-            // user has asked for it in settings.
-            return
-        }
-        UIPasteboard.general.asyncURL().uponQueue(.main) { res in
-            guard let copiedURL: URL? = res.successValue,
-                let url = copiedURL else {
-                return
-            }
-
-            let absoluteString = url.absoluteString
-
-            guard self.shouldDisplayBar(absoluteString) else {
-                return
-            }
-
-            self.lastDisplayedURL = absoluteString
-
-            self.clipboardToast =
-                ButtonToast(
-                    labelText: .GoToCopiedLink,
-                    descriptionText: url.absoluteDisplayString,
-                    buttonText: .GoButtonTittle,
-                    completion: { buttonPressed in
-                        if buttonPressed {
-                            self.delegate?.settingsOpenURLInNewTab(url)
-                        }
+        // There's no point in doing any of this work unless the
+        // user has asked for it in settings.
+        guard self.prefs.boolForKey("showClipboardBar") ?? false else { return }
```

Comment:
Do we need all the `.self` in this function?

Reply:
no its not needed, I will remove the other in the function too